modules/nf-core/gatk4/haplotypecaller/main.nf

```diff
@@ -9,11 +9,11 @@ process GATK4_HAPLOTYPECALLER {
 
     input:
     tuple val(meta),  path(input), path(input_index), path(intervals), path(dragstr_model)
-    tuple val(meta2), path(fasta)
-    tuple val(meta3), path(fai)
-    tuple val(meta4), path(dict)
-    tuple val(meta5), path(dbsnp)
-    tuple val(meta6), path(dbsnp_tbi)
+    tuple val(ref_meta), path(fasta)
+    tuple val(ref_idx_meta), path(fai)
+    tuple val(ref_dict_meta), path(dict)
+    tuple val(dbsnp_meta), path(dbsnp)
+    tuple val(dbsnp_tbi_meta), path(dbsnp_tbi)
```

Comment:
Like the idea, but not sure about its implementation, I'd reckon it's something we should talk more about it

Reply:
with things like multi-reference mapping, this kind of approach becomes necessary, to have a way to associate the correct fai, dict, dbsnp to the correct reference. It's something I've implemented in other nf-core modules without issue ( https://github.com/nf-core/modules/pull/3660 ).
Is there a reason not to do it like this?

Reply:
sorry wasn't clear at all.

I like the `meta5` -> `dbsnp_meta` change, just wondering if `meta_dbsnp` would be better, and if we can agree on a guideline

Reply:
oh! I see. It is following the suggestion of @mashehu here https://github.com/nf-core/modules/pull/4582#discussion_r1424019809
Just trying to keep the changes I end up making across different modules a bit consistent :D I'm all for having a convention on this for sure.